src/resources/lang/zh-Hant/crud.php

```diff
@@ -39,10 +39,15 @@
     'to'                   => '到',
     'undo'                 => '回復',
     'revision_restored'    => '成功回復修訂版本',
+    'guest_user'           => '遊客',
+
+    // Translatable models
+    'edit_translations' => 'EDIT TRANSLATIONS',
```

Comment:
Why is this still untranslated ?

Reply:
It's rather ironic that `edit translations` is the one that is **not** translated if I may say so myself 🙄

Reply:
@lloy0076 - yep, my point exactly:)

Reply:
i miss that part :P
updated.

Reply:
Haha. "edit_translations", out of all strings :-) Nice. Thanks guys!